sympy/physics/control/lti.py

```diff
@@ -1719,37 +1748,41 @@ class Feedback(Basic):
     To negate a ``Feedback`` object, the ``-`` operator can be prepended:
 
     >>> -F1
-    Feedback(TransferFunction(-3*s**2 - 7*s + 3, s**2 - 4*s + 2, s), TransferFunction(5*s - 10, s + 7, s))
+    Feedback(TransferFunction(-3*s**2 - 7*s + 3, s**2 - 4*s + 2, s), TransferFunction(10 - 5*s, s + 7, s), -1)
     >>> -F2
-    Feedback(Series(TransferFunction(-1, 1, s), TransferFunction(2*s**2 + 5*s + 1, s**2 + 2*s + 3, s), TransferFunction(5*s + 10, s + 10, s)), TransferFunction(1, 1, s))
+    Feedback(Series(TransferFunction(-1, 1, s), TransferFunction(2*s**2 + 5*s + 1, s**2 + 2*s + 3, s), TransferFunction(5*s + 10, s + 10, s)), TransferFunction(-1, 1, s), -1)
 
     See Also
     ========
 
-    TransferFunction, Series, Parallel
+    MIMOFeedback, Series, Parallel
 
     """
-    def __new__(cls, num, den):
-        if not (isinstance(num, (TransferFunction, Series))
-            and isinstance(den, (TransferFunction, Series))):
-            raise TypeError("Unsupported type for numerator or denominator of Feedback.")
-
-        if num == den:
-            raise ValueError("The numerator cannot be equal to the denominator.")
-        if not num.var == den.var:
-            raise ValueError("Both numerator and denominator should be using the"
+    def __new__(cls, sys1, sys2=None, sign=-1):
+        if not sys2:
+            sys2 = TransferFunction(1, 1, sys1.var)
+
+        if not (isinstance(sys1, (TransferFunction, Series))
+            and isinstance(sys2, (TransferFunction, Series))):
+            raise TypeError("Unsupported type for `sys1` or `sys2` of Feedback.")
+
+        if sign not in [-1, 1]:
+            raise ValueError("Unsupported type for feedback. `sign` arg should "
+                "either be 1 (positive feedback loop) or -1 (negative feedback loop).")
+
+        if Mul(sys1.to_expr(), sys2.to_expr()).simplify() == sign:
+            raise ValueError("The equivalent system will have zero denominator.")
+
+        if sys1.var != sys2.var:
+            raise ValueError("Both `sys1` and `sys2` should be using the"
                 " same complex variable.")
-        obj = super().__new__(cls, num, den)
-        obj._num = num
-        obj._den = den
-        obj._var = num.var
 
-        return obj
+        return super().__new__(cls, sys1, sys2, _sympify(sign))
 
     @property
-    def num(self):
+    def sys1(self):
         """
-        Returns the primary plant of the negative feedback closed loop.
+        Returns the feedforward system of the closed-loop feedback.
```

Comment:
```suggestion
        Returns the feedforward system of the feedback interconnection.
```
I'd refrain from using "closed-loop" in all the docstrings.